src/app/wizard/step/provider-settings/provider/extended/kubevirt/template.html

```diff
@@ -97,15 +101,17 @@
                 <mat-error *ngIf="control.get(Controls.PreAllocatedDataVolumeUrl).hasError( 'required')">
                   <strong>Required</strong>
                 </mat-error>
+                <mat-error *ngIf="control.get(Controls.PreAllocatedDataVolumeUrl).hasError( 'pattern')">
+                  Operating System URL should comply with <b>lowercase RFC 1123</b>
+                </mat-error>
               </mat-form-field>
             </div>
           </div>
         </div>
       </div>
     </div>
 
-    <div *ngIf="!preAllocatedDataVolumesFormArray.length"
-         class="km-text-muted">
+    <div class="km-text-muted">
```

Comment:
IMO, it should be in TOP when any new Local Disk configurations are added. Otherwise it will keep sliding down ... which IMHO is not the correct UI flow? thoughs?

![placement](https://user-images.githubusercontent.com/17727069/199424010-697798cf-8cf2-4531-ac16-b5070518cdfd.png)

Reply:
this question would be better answered by @cschieder or @mfranczy

Reply:
I agree with @KhizerRehan

Reply:
looks good?
<img width="2539" alt="image" src="https://user-images.githubusercontent.com/56511988/199469271-22bc8351-db73-4474-af5f-72484975c5b3.png">

Reply:
looks great to me.